_episodes/01-quickstart.md

```diff
@@ -2,6 +2,8 @@
 title: "Quickstart guide"
 teaching: 2
 exercises: 8
+compatibility: ESMValTool v.2.6.0
+
```

Comment:
Please remove line 35 in this lesson where it is mentioned `> This quickstart guide is compatible with ESMValTool v2.6.0.`

Reply:
Done.